api/src/main/java/net/md_5/bungee/api/plugin/PluginManager.java

```diff
@@ -295,12 +295,11 @@ private boolean enablePlugin(Map<PluginDescription, Boolean> pluginStatuses, Sta
         // do actual loading
         if ( status )
         {
-            try
-            {
-                URLClassLoader loader = new PluginClassloader( new URL[]
```

Comment:
Doing otherwise will lead to issues in nearly every plugin having more than 1 class, since a try-with-resources will close the resource although it was not closed afterwards before for a reason.